src/Prismic/ApiFactory.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Prismic;
+
+class ApiFactory
+{
+    public function get(...$args): Api
```

Comment:
Note: we could also force the parameters as in the target method. In some way it would be cleaner actually :thinking: